modules/administration/pages/channel-management.adoc

```diff
@@ -0,0 +1,37 @@
+[[channel-management]]
+= Channel Management
+// Antora entities
+ifndef::backend-pdf[]
+include::ROOT:partial$entities.adoc[]
+endif::[]
+// PDF entities
+ifdef::backend-pdf[]
+include::_partials/entities.adoc[]
+endif::[]
+
+Channels are a method of grouping software packages.
+
+In {productname}, channels are grouped into base and child channels, with base channels usually grouped by hardware architecture and software version, and child channels being related to a specific base channel.
+
+While {productname}  provides all required channels, you might find it useful to create custom channels specific to your environment.
+{productname} Administrators and Channel Administrators have channel management authority, which gives them the ability to create and manage their own custom channels.
+
+This section gives more detail on how to create, administer, and delete custom channels.
+
+== Create Custom Channels
+
+TODO:: write this
+
+Note that if you include a package tagged with [systemitem]``release`` in your channel, then it will be automatically installed when the channel is installed.
+
+== Administer Custom Channels
+
+TODO:: write this
+
+== Delete Custom Channels
+
+TODO:: write this
+
+Note that when channels are deleted, the packages that are part of the deleted channel are not automatically removed.
+If you try to update packages that have had their channel deleted, it can cause errors.
+In this case, you will need to manually remove the packages.
```

Comment:
I would mention it's necessary to set those package to "removed" in package state page. (States > Package on the system page.) Simple removal of the package by zypper or thru "Software" menu doesn't solve the problem as package installation is still enforced by state.